src/defaultConfig.ts

```diff
@@ -81,6 +81,7 @@ export const defaultSecurityConfig = (serverlUrl: string): ModuleOptions => ({
     exclude: [/node_modules/, /\.git/]
   },
   ssg: {
-    hashScripts: true
+    hashScripts: true,
+    hashStyles: false
```

Comment:
suggestion. I think that hash styles will be introduced in another PR. Could you please revert it so that we can only ship the cheerio addition with this PR? :) 

The hash styles I was hoping to release in the next RC version.

Reply:
Done!